lampo-common/src/conf.rs

```diff
@@ -213,6 +215,8 @@ impl TryFrom<String> for LampoConf {
             _ => "info".to_string(),
         };
         let log_file = conf.get_conf("log-file").unwrap_or_else(|_| None);
+        let alias = conf.get_conf("alias").unwrap_or(None);
```

Comment:
we need to announce this, but looks like I still need to finish some work here https://github.com/vincenzopalazzo/lampo.rs/pull/202

Lets open a issue and keep track of this

Reply:
>I don't get it. Why do we need to announce alias? Is this related to bolt12?

we need to announce to the network with a node announcement, it is not related to bolt12, but I did some of the work on the PR that support the bolt12